pkg/sql/catalog/systemschema/system.go

```diff
@@ -1338,10 +1338,11 @@ CREATE TABLE public.inspect_errors (
     id OID NOT NULL,
     primary_key STRING NULL,
     details STRING NOT NULL,
+    crdb_internal_expiration TIMESTAMPTZ NOT NULL DEFAULT current_timestamp():::TIMESTAMPTZ + '90 days':::INTERVAL ON UPDATE current_timestamp():::TIMESTAMPTZ + '90 days':::INTERVAL,
     CONSTRAINT "primary" PRIMARY KEY (error_id ASC),
     INDEX object_idx (id ASC),
-	FAMILY "primary" (error_id, job_id, error_type, database_id, schema_id, id, primary_key, details)
-);`
+	FAMILY "primary" (error_id, job_id, error_type, database_id, schema_id, id, primary_key, details, crdb_internal_expiration)
```

Comment:
this is using the default `ttl_job_cron` setting which is `@daily`. considering that we expect corruptions to be pretty rare, that seems unnecessarily frequent. how about we use weekly instead? curious if @spilchen has thoughts on this.